lib/content-services/src/lib/category/categories-management/categories-management.component.scss

```diff
@@ -11,6 +17,20 @@
         .adf-btn-padded {
             margin-right: -14px;
         }
+
+        .mat-form-field-underline {
+            display: none;
```

Comment:
I was already asking about this, why do you change this?

Reply:
![underline_none](https://github.com/Alfresco/alfresco-ng2-components/assets/107834418/0c303956-c6a0-481d-ac53-329cbb39f373)
Using mat-form-field, as per Figma we should not display the underline, so I added display none.
here is the Figma link: https://www.figma.com/file/gtkH6ZeJAmoEx4g43cQdke/Bulk-Selection-and-Edit-Metadata-Properties?node-id=5610%3A92242&mode=dev

Reply:
got it, thanks